x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_creation_ui/hooks/use_esql_index.test.ts

```diff
@@ -33,6 +33,6 @@ describe('useEsqlIndex', () => {
 
     const { result } = renderHook(() => useEsqlIndex(typeErrorCausingQuery, 'esql'));
 
-    expect(result.current).toEqual(['auditbeat*']);
```

Comment:
It's interesting that the grammar now identifies the other index! In my testing this appears to only happen for a few single-character typos; most of the time the behavior remains as it was.

For that reason I don't think we should extend the tests to document the new behavior, and this change makes sense 👍